extensions/wikia/BannerNotifications/js/BannerNotifications.js

```diff
@@ -171,11 +170,12 @@
 	 * Called once to instantiate this feature
 	 */
 	function init() {
+		$header = $('#globalNavigation');
+		
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ATrailingWhitespace)